src/PowerShell/Help/Microsoft.WinGet.Client/Disable-WinGetSetting.md

```diff
@@ -0,0 +1,87 @@
+﻿---
+external help file: Microsoft.WinGet.Client.Cmdlets.dll-Help.xml
+Module Name: Microsoft.WinGet.Client
+ms.date: 08/01/2024
+online version:
+schema: 2.0.0
+title: Disable-WinGetSetting
+---
+
+# Disable-WinGetSetting
+
+## SYNOPSIS
+Disables an administrative setting.
+
+## SYNTAX
+
+```
+Disable-WinGetSetting [-Name] <String> [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This command disables an administrative setting. The command must be run from an elevated session
+using the **Run as Administrator** option.
+
+This command support the following administrative settings:
+
+- LocalManifestFiles
+- BypassCertificatePinningForMicrosoftStore
+- InstallerHashOverride
+- LocalArchiveMalwareScanOverride
+- ProxyCommandLineOptions
+- DefaultProxy
```

Comment:
```suggestion
```

Needs to be removed in Disable cmdlet as well

Reply:
Will address this in another PR